pom.xml

```diff
@@ -35,6 +35,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>3.2.0</version>
+                <configuration>
+                    <source>8</source>
+                </configuration>
```

Comment:
What's this?

Reply:
This is to fix a javadoc compiling error on java 14